packages/sdk/src/types/runtime/collider.ts

```diff
@@ -24,8 +30,11 @@ export class Collider implements ColliderLike {
     // public collisionLayer = CollisionLayer.Object;
 
     // Readonly params that are not patchable or observable.
-    // tslint:disable-next-line:variable-name
+    // tslint:disable:variable-name
     private _colliderGeometry: Readonly<ColliderGeometry>;
+    private _collisionHandlers: CollisionHandlers = {};

```

Comment:
You can do all this with a single EventEmitter. I think an EventEmitter is more appropriate here, since you can have multiple listeners registered for an event.